src/components/Feed.tsx

```diff
@@ -2,16 +2,32 @@ import React, { useState } from "react";
 import NewPost from "./NewPost";
 import PostList from "./PostList";
 import { Button } from "antd";
-import { useAppSelector } from "../redux/hooks";
+import { useAppDispatch, useAppSelector } from "../redux/hooks";
+import * as types from "../utilities/types";
+import { ArrowLeftOutlined } from "@ant-design/icons";
+import { setDisplayedProfileId } from "../redux/slices/userSlice";
 
 enum FeedTypes {
   MY_FEED,
   MY_POSTS,
   DISCOVER,
+  PROFILE_POSTS,
 }
 
 const Feed = (): JSX.Element => {
+  const dispatch = useAppDispatch();
+
   const userId = useAppSelector((state) => state.user.id);
+  const currentProfileId = useAppSelector(
```

Comment:
I want to figure out the best naming for this and use it throughout. if user.id = userId, user.displayedProfileId = ?